tools/clang/test/CodeGenSPIRV_Lit/rich.debug.scope.after.compound.statement.hlsl

```diff
@@ -20,24 +20,24 @@ VS_OUTPUT main(float4 pos : POSITION,
   float x = a + b + c;
   {
 //CHECK:      DebugScope [[bb1]]
-//CHECK-NEXT: OpLine [[file:%\d+]] 24
+//CHECK-NEXT: OpLine [[file:%[0-9]+]] 24
     float a = 3.0;
     float b = 4.0;
     x += a + b + c;
     {
 //CHECK:      DebugScope [[bb2]]
-//CHECK-NEXT: OpLine [[file:%\d+]] 32
-//CHECK-NEXT: OpStore [[var_a:%\w+]] %float_6
+//CHECK-NEXT: OpLine [[file_0:%[0-9]+]] 32
```

Comment:
Labels cannot be redefined in real LIT tests?
(Or is that frowned upon?)

Reply:
llvm FileCheck will report error on case like this.
https://github.com/microsoft/DirectXShaderCompiler/blob/main/tools/clang/test/CodeGenSPIRV/fn.param.isomorphism.hlsl#L103

// CHECK-NEXT: [[arr_0:%\d+]] = OpLoad %int [[arr]]
// CHECK-NEXT: [[arr_0:%\d+]] = OpIAdd %int [[arr_0]] %int_1
arr_0 is redefined but here we expect the [[arr_0]] still match the old define.

Reply:
Right, but here redefining it would be fine no?
(I'm fine with not redefining, just curious)

Reply:
Yes, noted, thanks 😊